civis/tests/test_io.py

```diff
@@ -398,6 +398,147 @@ def test_civis_file_to_table_table_doesnt_exist(
             **expected_kwargs
         )
 
+    @pytest.mark.civis_file_to_table
+    @mock.patch('civis.io._tables._process_cleaning_results')
+    @mock.patch('civis.io._tables._run_cleaning')
+    def test_civis_file_to_table_table_exists_all_sql_types_missing(
+            self,
+            m_run_cleaning,
+            m_process_cleaning_results,
+            _m_get_api_spec
+    ):
+        table = "scratch.api_client_test_fixture"
+        database = 'redshift-general'
+        mock_file_id = 1234
+        mock_cleaned_file_id = 1235
+        mock_import_id = 8675309
+
+        self.mock_client.imports.post_files_csv.return_value\
+            .id = mock_import_id
+        self.mock_client.get_database_id.return_value = 42
+        self.mock_client.default_credential = 713
+
```

Comment:
Here we want to mock the api client response for `self.mock_client.get_table_id` based on whether or not the table exists. If the table exists, it should look like [this](https://github.com/civisanalytics/civis-python/blob/6d632e4e71888bdc63aba7655442bdc715c5f04d/civis/tests/test_io.py#L224). If the table does not exist then it should look like [this](https://github.com/civisanalytics/civis-python/blob/6d632e4e71888bdc63aba7655442bdc715c5f04d/civis/tests/test_io.py#L322).

Reply:
Yeah I see now, since I had the logic statement wrong I miswrote the tests

Reply:
done